collectors/plugins.d/pluginsd_parser.c

```diff
@@ -560,6 +560,71 @@ PARSER_RC pluginsd_overwrite(char **words, void *user, PLUGINSD_ACTION  *plugins
     return PARSER_RC_OK;
 }
 
+PARSER_RC pluginsd_guid(char **words, void *user, PLUGINSD_ACTION *plugins_action)
+{
+    char *uuid_str = words[1];
+    uuid_t uuid;
+
+    if (unlikely(!uuid_str)) {
+        error("requested a GUID, without a uuid.");
+        return PARSER_RC_ERROR;
+    }
+    if (unlikely(strlen(uuid_str) != GUID_LEN || uuid_parse(uuid_str, uuid) == -1)) {
+        error("requested a GUID, without a valid uuid string.");
+        return PARSER_RC_ERROR;
+    }
+
+    info("Parsed uuid=%s", uuid_str);
```

Comment:
Change to `debug` or remove.

Reply:
Will do, but I need them in info for a little while to help with debugging. Same deal as the ACLK prints that remained `info()`. I will change this to `debug` as soon as I'm done testing compaction.